Introduce multi host in pulsar doc (#4546)
apache/pulsar: site2/docs/deploy-bare-metal.md

> However if you don't have enough machines, or are trying out Pulsar in cluster mode (and expand the cluster later),
> you can even deploy Pulsar in one node, where it will run zookeeper, bookie and broker in same machine.

> > If you don't have a DNS server, you can use multi host in service url to replacing the DNS.

Reviewer: ```suggestion
> If you don't have a DNS server, you can use multi-host in service URL instead.
```
